src/components/new-safe/create/useSyncSafeCreationStep.ts

```diff
@@ -5,29 +5,17 @@ import type { NewSafeFormData } from '@/components/new-safe/create/index'
 import useWallet from '@/hooks/wallets/useWallet'
 import useIsWrongChain from '@/hooks/useIsWrongChain'
 import { useRouter } from 'next/router'
-import { AppRoutes } from '@/config/routes'
 
 const useSyncSafeCreationStep = (setStep: StepRenderProps<NewSafeFormData>['setStep']) => {
-  const [safeAddress, pendingSafe] = useUndeployedSafe()
+  const [_, pendingSafe] = useUndeployedSafe()
```

Comment:
Is `pendingSafe` still needed here?

Reply:
Good spot. Strange that no linter complained about the unused dependency in the hook dep array.

I will remove it